tests/admin_scripts/tests.py

```diff
@@ -61,7 +61,8 @@ def write_settings(self, filename, apps=None, is_dir=False, sdict=None, extra=No
                 'ROOT_URLCONF',
                 'SECRET_KEY',
                 'TEST_RUNNER',  # We need to include TEST_RUNNER, otherwise we get a compatibility warning.
```

Comment:
Actually 'TEST_RUNNER'/'MIDDLEWARE_CLASSES' can be removed as the compatibility checks mentioned are removed. I've done this on master so please rebase your branch.